### ISSUE TITLE
Bottom Bar is Duplicated

### ISSUE THREAD

**Issue by Brian-Kaplan** [zbtang/React-Native-TextInputLayout#4]
When I install your demo app it works fine but, when attempting to integrate it into my own it does not. See in the image below the bottom bar of the text is duplicated...?

```
  render() {
    return (
      <View style={styles.container}>
        <TextInputLayout style={styles.inputLayout}>
          <TextInput
            style={styles.textInput}
            placeholder={'Password'}
            secureTextEntry={true}
          />
      </TextInputLayout>
    </View>
    );
  }
});

const styles = StyleSheet.create({
  container: {
    flex: 1
  },
  textInput: {
    fontSize: 16,
    marginTop: 20,
    height: 40
  },
  inputLayout: {
    marginTop: 16,
    marginHorizontal: 36
  }
});
```

![image](https://cloud.githubusercontent.com/assets/11176781/21996688/7804889c-dbf9-11e6-88f2-1556cdf21f9c.png)

**Comment by Brian-Kaplan** [comment]
Are you somehow removing the bottom bar that comes with android EditText or are you just covering it with a react native <View> component?

**Comment by Brian-Kaplan** [comment]
Can confirm that this issue is related to react/react native version. 

Your Versions:
    "react": "15.2.1",
    "react-native": "0.30.0",
    "rn-textinputlayout": "^1.0.0",

My Versions:
    "react": "15.4.1",
    "react-native": "0.38.0",
    "rn-textinputlayout": "^1.0.1",

**Comment by zbtang** [comment]
Hello, this bug has been fixed on latest version 1.0.4, thanks!
